tcms/testruns/static/testruns/js/get.js

```diff
@@ -231,18 +233,27 @@ function renderTestExecutionRow(template, testExecution, testExecutionStatus) {
 /////// the functions below were used in bulk-menu actions
 /////// and need updates before they can be used again
 ///////
-function changeExecutionAssignee() {
-    const executions = 0 // todo: this is the list of all selected executions
-    if (!executions.length) {
-        window.alert(default_messages.alert.no_case_selected);
-        return false;
+function changeAssigneeBulk() {
+    const checkboxes = selectedCheckboxes()
+    if (!checkboxes.length) {
+        return false
     }
-    //todo: all texts in alert/prompt must come translated from the HTML template
-    var assignee = window.prompt('Please type new email or username for assignee');
+
+    const testExecutionIds = []
+    checkboxes.each((_index, checkbox) => {
+        const testExecutionId = $(checkbox).data('test-execution-id')
+        testExecutionIds.push(testExecutionId)
+    })
```

Comment:
You can move this one in the selectedCheckboxes too and return a more complex data structure. Otherwise will have to repeat it in every bulk handler. Or make a new function which returns the IDs based on the passed selection.